platforms/documentation/docs/src/snippets/bestPractices/useCacheabilityAnnotations/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+// tag::avoid-this[]
+tasks.register<Delete>("clean") {
```

Comment:
is it necessary to define clean task?

Reply:
Yes.  This task is not defined by default, and there are no plugins used to add rule-based tasks like "cleanAddBad1".